providers/pipedrive/objectNames.go

```diff
@@ -0,0 +1,12 @@
+package pipedrive
+
+import (
+	"github.com/amp-labs/connectors/common/handy"
+)
+
+// ObjectNameToResponseField maps ObjectName to the response field name which contains that object.
+var ObjectNameToResponseField = handy.NewDefaultMap(map[string]string{}, //nolint:gochecknoglobals
```

Comment:
This declaration is not needed, for OpenAPI tool please use: `api3.IdenticalObjectCheck` which does exactly as the method.
Ex:
```
objects, err := explorer.GetBasicReadObjects(
	ignoreEndpoints, nil, nil, api3.IdenticalObjectCheck,
)
```